### PR TITLE
SC-1928 Onboarded Question and Question Set APIs

### PR DESCRIPTION
**Pull request by rheafernandes**

### CODE REVIEW

**Review by amitpriyadarshi — APPROVED** [review]

File: ansible/roles/kong-api/defaults/main.yml [hunk 1]
```diff
@@ -5692,3 +5695,255 @@ kong_apis:
...
+    uris: "{{ question_prefix }}/v1/review"
+    upstream_url: "{{ assessment_service_url }}/question/v4/review"
+    strip_uri: true
+    plugins:
+      - name: jwt
+      - name: cors
+      - "{{ statsd_pulgin }}"
+      - name: acl
+        config.whitelist:
+          - 'contentCreate'
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5781]: change this to contentAdmin

File: ansible/roles/kong-api/defaults/main.yml [hunk 2]
```diff
@@ -5692,3 +5695,255 @@ kong_apis:
...
+    uris: "{{ questionset_prefix }}/v1/review"
+    upstream_url: "{{ assessment_service_url }}/questionset/v4/review"
+    strip_uri: true
+    plugins:
+      - name: jwt
+      - name: cors
+      - "{{ statsd_pulgin }}"
+      - name: acl
+        config.whitelist:
+          - 'contentCreate'
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5889]: please change it to contentAdmin

File: ansible/roles/kong-api/defaults/main.yml [hunk 3]
```diff
@@ -5692,3 +5695,255 @@ kong_apis:
...
+      - name: rate-limiting
+        config.policy: local
+        config.hour: "{{ medium_rate_limit_per_hour }}"
+        config.limit_by: credential
+      - name: request-size-limiting
+        config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: questionSetPublish
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5897]: will the same user who creates the question set publishes it or an admin, based on that ACL need to be updated!

File: ansible/roles/kong-api/defaults/main.yml (hunk 1 above)
Comment (rheafernandes) [reply ansible/roles/kong-api/defaults/main.yml]: Changes done

File: ansible/roles/kong-api/defaults/main.yml (hunk 2 above)
Comment (rheafernandes) [reply ansible/roles/kong-api/defaults/main.yml]: Changes done

File: ansible/roles/kong-api/defaults/main.yml (hunk 3 above)
Comment (rheafernandes) [reply ansible/roles/kong-api/defaults/main.yml:5897]: Changed it

**Review by SMYALTAMASH — APPROVED** [review]

**Pull request merged**